projects/dynamicforms/src/components/upload/upload.component.ts

```diff
@@ -62,6 +65,32 @@ export class UploadComponent extends AbstractFormComponent implements OnInit {
   ngOnInit() {
     super.ngOnInit();
     this.uploadControl = this.group.get(this.config.name);
+    this.subscription.add(
```

Comment:
Maybe we could have put this inside separate method and then call it here?

Reply:
Of course, it will be easier to override.